urban-dashboard/src/components/mapboxViewer/index.jsx

```diff
@@ -24,7 +24,7 @@ export class MapBoxViewer extends Component {
 
     plotMap() {
         mapboxgl.accessToken = accessToken;
-        let mapboxStyleUrl = 'mapbox://styles/mapbox/satellite-v9';
+        let mapboxStyleUrl = 'mapbox://styles/covid-nasa/cldu1cb8f00ds01p6gi583w1m';
```

Comment:
Take this from env

Reply:
Is this the style for 3d or 2d?

Reply:
this is for the city labels and the one that the dashboard uses
it supports 3d, but by setting projection to mercator, we can make it 2d